### PR TITLE
github provider: allow multiple teams

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

**Comment by jehiah** [comment]
@raphink this looks good; Would you be able to add some docs for this functionality?

**Comment by raphink** [comment]
@jehiah sure thing, just added it.

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The Azure AD auth provider uses `openid` as it default scope. It uses `https://g
 The GitHub auth provider supports two additional parameters to restrict authentication to Organization or Team level access. Restricting by org and team is normally accompanied with `--email-domain=*`
 
     -github-org="": restrict logins to members of this organisation
-    -github-team="": restrict logins to members of this team
+    -github-team="": restrict logins to members of this team, multiple values can be entered, separated by a comma
```
Comment (jehiah) [inline README.md]: Can you specifically mention that any single team match is sufficient for authorization? (as opposed to requiring all team matches)

Comment (raphink) [inline README.md]: I changed the wording, tried to avoid making it too long. Is that clear enough?

Comment (jehiah) [inline README.md]: :+1:

**Pull request merged**